apps/fxc-front/src/app/redux/track-slice.ts

```diff
@@ -146,8 +153,10 @@ export const fetchTrack = createAsyncThunk('track/fetch', async (params: FetchTr
   if (route && route.alt.length > 0) {
     const coords = [];
     for (let i = 0; i < route.alt.length; i++) {
+      // @ts-ignore funky error: google name not found
```

Comment:
that's because the google namespace only exists after the Maps API is loaded.
It should only be used when we are sure it is loaded.